### PR TITLE
feat: add PGN Mainnet+Testnet and Arthera Testnet 1.3.0 L2 contracts

### PR DESCRIPTION
**Pull request by bertux**
Please add PGN Mainnet+Testnet and Arthera Testnet based on:
https://github.com/safe-global/safe-deployments/pull/315 done by @sodew for PGN
https://github.com/safe-global/safe-deployments/pull/329 in review for Arthera
Thank you!

### CODE REVIEW

**Comment by bertux** [comment]
I have read the CLA Document and I hereby sign the CLA

File: gnosis/safe/addresses.py [hunk 1]
```diff
@@ -595,6 +595,12 @@
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 2362236, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 2362238, "1.3.0"),
     ],
+    EthereumNetwork.PGN_MAINNET: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 344345, "1.3.0+L2"),
+    ],
```
Comment (moisses89) [inline gnosis/safe/addresses.py:600]: 1.3.0 contract was not deployed there? 0x3E5c63644E683549055b9Be8653de26E0B4CD36E

File: gnosis/eth/ethereum_network.py [hunk 2]
```diff
@@ -736,6 +736,9 @@ class EthereumNetwork(Enum):
     ZENIQ = 383414847825
     PDC_MAINNET = 666301171999
     MOLEREUM_NETWORK = 6022140761023
+    PGN_MAINNET = 424
+    PGN_TESTNET = 58008
+    ARTHERA_TESTNET = 10243
```
Comment (moisses89) [inline gnosis/eth/ethereum_network.py:741]: This should be sorted by chain_id.

**Review by moisses89 — CHANGES_REQUESTED** [review]

File: gnosis/safe/addresses.py (hunk 1 above)
Comment (bertux) [reply gnosis/safe/addresses.py:600]: I've followed the README instructions, and from what I understood I should only fill 1.3.0 L2 version contract address and not 1.3.0 contract address.
So it seems I misunderstood, I will add the 1.3.0 contract address too.

**Comment by sodew** [comment]
Changes fixed. Please review @moisses89

File: gnosis/safe/addresses.py (hunk 1 above)
Comment (sodew) [reply gnosis/safe/addresses.py:643]: @moisses89 added 1.3.0 contract

File: gnosis/eth/ethereum_network.py (hunk 2 above)
Comment (sodew) [reply gnosis/eth/ethereum_network.py]: @moisses89 sorted by chain_id

**Review by moisses89 — APPROVED** [review]

**Comment by moisses89** [comment]
This PR should be merged first https://github.com/safe-global/safe-deployments/pull/329

**Review by moisses89 — APPROVED** [review]

**Pull request merged**